boards/arm/nrf9160_pca10090/doc/nrf9160_pca10090.rst

```diff
@@ -0,0 +1,207 @@
+.. _nrf9160_pca10090:
+
+nRF9160-PCA10090
+################
+
+Overview
+********
+
+Zephyr applications use the nrf9160_pca10090 board configuration
+to run on the nRF9160 Development Kit (PCA10090) hardware. The
```

Comment:
The official name is "nRF9160 DK" (not Development Kit).

Reply:
Thanks @ru-fu, that is now fixed.